htdocs/PI/write/PIWriteRequest.php

```diff
@@ -85,6 +89,12 @@ public function __construct() {
         $configServ = new config();
         $this->baseUrl = $configServ->getServerBaseUrl() . "/gocdbpi";
         $this->docsURL = $configServ->getWriteApiDocsUrl();
+
+        #Define some generic exception messages (remaining ones will be generated once entity type and value are known)
+        $this->genericExceptionMessages["URLFormat"] =
+          "API requests should take the form $this->baseUrl" .
```

Comment:
Does this render correctly? I though more complex references to variables in strings has to be wrapped in `{}`, but maybe PHP is better now? :confused:

Reply:
Yep 
![image](https://user-images.githubusercontent.com/4125374/67509017-02898f00-f68a-11e9-9edf-925f3f8732d7.png)